datadog/dogstatsd/base.py

```diff
@@ -225,7 +235,7 @@ def get_socket(self):
 
         return self.socket
 
-    def open_buffer(self, max_buffer_size=50):
+    def open_buffer(self):
```

Comment:
This breaks the current API FWIW.
I'm not against it, but if you're going to break the API, it'd be better to provide a new one that e.g. fixes #439

Reply:
I had a quick discussion about that, we agreed not to break the API, pending upcoming rework (I keep in mind the redis pipeline idea thanks again BTW).